src/payment/strategies/mollie/mollie-initialize-options.ts

```diff
@@ -1,7 +1,9 @@
 export default interface MolliePaymentInitializeOptions {
+    containerId: string;
```

Comment:
can you provide a docblock explaining what each attribute does? It's unclear to me why `containerId` is now a required property and before it wasnt